pages/1.13/security/ent/ldap/ldap-auth/index.md

```diff
@@ -9,70 +9,151 @@ enterprise: true
 <!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
 
 
-In this section you will set up the authentication method and parameters for your LDAP directory.
+In this section you will set up the authentication method and parameters for your LDAP directory. There are two bind types and two authentication methods you can use.
 
-# Selecting the authentication method
+# Bind Types
 
-1. Once you have finished specifying your connection parameters in the **Connection** tab, click **Authentication**.
+Bind operations are used to authenticate clients to the directory server,  establish an authorization identity that will be used for subsequent operations processed on that connection, and to specify the LDAP protocol version that the client will use.
 
-1.  Provide the full DN of the user account you will use to connect to the LDAP server to import users, groups, and check user credentials in the **Lookup DN** field. A few examples follow.
+There are two bind types available to you: 
 
-        cn=read-only-user,dc=los-pollos,dc=io
-        uid=read-only-user,cn=users,cn=accounts,dc=demo1,dc=freeipa,dc=org
-        uid=read-only-user,ou=users,dc=example,dc=com
+- Anonymous Bind - Uses a simple bind DN and a user DN template. Anyone can connect to the LDAP server. This bind type does not require a **Lookup DN** or **Lookup Password**. 
+- LDAP Credentials - Requires a **Lookup DN** and **Lookup Password**. LDAP credentials must be previously established.
 
-  **Tip:** We recommend a read-only user account.
+# Authentication methods
 
-1. Provide the password of the account in the **Lookup Password** field.
+There are two authentication methods available to you.
+
+- Simple bind. Authentication by simple bind is the most common way to authenticate an LDAP client. You can either bind Anonymously (using a simple bind DN) or you can provide LDAP credentials such as a lookup DN and lookup password. In either case, you must provide a user DN template. Choose Simple bind if your LDAP user name is part of your [distinguished name (DN)](https://www.ldap.com/ldap-dns-and-rdns).     
```

Comment:
It seems this is combining authentication method with bind type. Anonymous bind type != simple bind authentication method. You could choose anonymous bind type with search bind authentication method.

Reply:
Got it. I have revised the text.